frontend/src/pages/QueryBuilder/QueryBuilder.tsx

```diff
@@ -52,10 +52,21 @@ interface RouterParams {
   id?: string;
 }
 
+interface QueryBuilderContextValue {
+  defaultValue: string;
+}
+
 type QueryBuilderProps = ActionProps & ReduxState & RouteComponentProps<RouterParams>;
 
 const OBSOLETE_COLUMNS_LOG_MESSAGE = 'Obsolete Columns'; // eslint-disable-line @typescript-eslint/naming-convention
 
+const queryBuilderValueContext: QueryBuilderContextValue = {
+  defaultValue: 'Basic',
+};
+
+export const QueryBuilderContext =
```

Comment:
From my observation, use of context for this task is totally not required, so please remove usage of this context you created

Reply:
Pardon

Reply:
It is not required, add the changes I have suggested and run, it  works, without using context. So the extra lines of code are not required.